src/components/Theme/src/default-theme.js

```diff
@@ -48,9 +47,11 @@ export default function defaultTheme() {
 			patterns: {
 				primary: {
 					variant: 'primary',
+					color: '@colors.primary',
 				},
 				secondary: {
 					variant: 'secondary',
+					color: '@colors.secondary',
```

Comment:
I would default this to paragraph color for the default theme and let website override it if necessary.